src/pharmpy/workflows/model_database/local_directory.py

```diff
@@ -145,15 +151,37 @@ def retrieve_file(self, name, filename):
             raise FileNotFoundError(f"Cannot retrieve {filename} for {name}")
 
     def get_model(self, name):
-        filename = name + self.file_extension
-        path = self.path / name / filename
+        extensions = ['.ctl', '.mod']
         from pharmpy.model import Model
 
-        model = Model.create_model(path)
```

Comment:
Do I understand correctly that this only worked with NONMEM models before anyway?